tf2onnx/optimizer/merge_duplicated_nodes_optimizer.py

```diff
@@ -57,7 +59,7 @@ def _del_nodes_if_duplicated(self, nodes_group, graph):
             unprocessed_node = []
             nodes_to_process = [nodes_group[0]]
             for node in nodes_group[1:]:
-                if node.attr == nodes_to_process[0].attr:
+                if self._are_attr_equal(node.attr, nodes_to_process[0].attr):
```

Comment:
could you give some comments here why "node.attr == nodes_to_process[0].attr" is not work? thanks.

Reply:
A Tensor attribute has more fields than just the values of it
If you use `==` to compare two tensors their names get compared as well. (the same is true if the attribute is a graph). And that's not what we want because you can create multiple tensors from the same static values. 
What we need is to compare only the tensors shape, type and raw data

Reply:
you mean if tensor's shape & type & raw data are same, then we treat them as duplicated even other attributes are different

Reply:
could you please give out a attribute that we can ignore? thanks a lot!

Reply:
i am thinking of modifying "if node.attr == nodes_to_process[0].attr:" to "if node.attr == nodes_to_process[0].attr **or is_same_const_node()**".

any thought, jeff  and @nbcsm?

Reply:
I think we should not restrict this comparison to const nodes only. If we are determining the equality of nodes based on their behavior (type, data, inputs, and outputs) then other informational fields should be stripped from this comparison